lib/ProductOpener/Config_off.pm

```diff
@@ -1634,6 +1634,12 @@ $options{sample_product_code} = "093270067481501";    # A good product for you -
 
 # Number of requests per minutes for the search API
 $options{rate_limit_search} = 10;
+# Number of requests per minutes for all facets for anonymous users
+$options{rate_limit_facet_unregistered} = 2;
```

Comment:
2 requests seem very small. e.g. you do a first one to explore by brands, you click on a brand, that's already 2 facets...

Reply:
I increased it to 5.